backend/service/k8s/clientset.go

```diff
@@ -26,26 +27,34 @@ type ClientsetManager interface {
 
 type ContextClientset interface {
 	k8s.Interface
+	Metrics() k8s_metrics.Interface
 	Namespace() string
 	Cluster() string
 }
 
-func NewContextClientset(namespace string, cluster string, clientset k8s.Interface) ContextClientset {
+func NewContextClientset(namespace string, cluster string, clientset k8s.Interface, metrics k8s_metrics.Interface) ContextClientset {
```

Comment:
i'd rather not change this signature

can you add an additional function that is NewContextClientSetWithMetrics or similar?

Reply:
Made the suggested change